src/Transformers/AbstractTransformer.php

```diff
@@ -220,18 +173,14 @@ public function getLink($model): string
 
     /**
      * Get a link to the relation
-     *
-     * @param TModel $model
```

Comment:
This was actually useful, lets bring it back

Reply:
Same here, `$model` can be any related model, so it's not a `TModel`.